### ISSUE TITLE
TypeError: Path must be a string

### ISSUE THREAD

**Issue by gearmobile** [adchsm/Slidebars#229]
I think, your plugin has too many errors and it does not work under npm/webpack.

### package.json
``` javascript
...
  "main": [
    "dist/slidebars.min.js",
    "dist/slidebars.min.css"
  ],
...
```
main.js:

``` javascript
'use strict';
var $ = require('jquery');
var slidebars = require('Slidebars');
```
In the console gives an error:

``` bash
[BS] Browser Connected: Firefox, version: 47.0
[13:34:51] gulp-notify: [TARS notification] 
JavaScript has not been processed
Look in the console for details.

Error details:
./markup/static/js/main.js
Module not found: Error: Cannot resolve module 'Slidebars' in /home/zencoder/Projects/javascript/slidebars/markup/static/js
...
```
But when I change the package.json in the following form:

```javascript
...
"main": "dist/slidebars.min.js",
...
```
The error in the console disappears.

### Constructor name should not start with a lowercase letter

```javascript
'use strict';
var $ = require('jquery');
var slidebars = require('Slidebars');

$( document ).ready(function () {
    var controller = new slidebars();
    controller.init();
});
```

Error in the console:

```bash
7:26  error    A constructor name should not start with a lowercase letter  new-cap
```

### Slidebars is not a constructor

When I change the configuration file `.eslintrc` and line `"new-cap": 2` to line `"new-cap": 1`, the error changes to a warning, of course.

But in the browser console error appears:

``` javascript
jQuery.Deferred exception: slidebars is not a constructor @http://localhost:3004/static/js/main.js:53:23
```

The last error I can not do anything, as I don't know what to do.

**Comment by adchsm** [comment]
Thanks for letting me know, i've checked it out need to add the css file in style not main. I'll push a fix through now.

**Comment by adchsm** [comment]
Should be fixed now with 2.0.2 https://github.com/adchsm/Slidebars/releases/tag/2.0.2

**Comment by eyscode** [comment]
@adchsm It seems like you published v2.0.2 with a different name, you named it "slidebars" instead of "Slidebars", that's why when you install your library through `npm install Slidebars` it donwloads v2.0.0. https://www.npmjs.com/search?q=slidebars

Also as far as I see there is no way to use your library with `require`, when the module is imported it returns an empty json, I think it's not prepared for CommonJS, isn't it? You may include `module.exports` or stuff like that.
